desc/io/input_reader.py

```diff
@@ -776,6 +778,166 @@ def write_desc_input(filename, inputs, header=""):  # noqa: C901 - fxn too compl
 
         f.close()
 
+    @staticmethod
+    def descout_to_input(  # noqa: C901 - fxn too complex
+        filename,
```

Comment:
`filename` and `inputs` are a bit confusing, since `inputs` itself is a filename. Might be better to make it `input_filename` and `output_filename`. 
Or better yet, change `inputs` to `eq` and allow it to also be an instance of the `Equilibrium` class? (or load it from a file if its path-like)

Reply:
There are other routines in input_reader with confusing terminology like this. Let me try something else.